plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```diff
@@ -335,6 +336,7 @@ private void writeBlobMultipart(BlobInfo blobInfo, InputStream inputStream, long
                 new Storage.BlobTargetOption[0];
             SocketAccess.doPrivilegedVoidIOException(
                     () -> client().create(blobInfo, buffer, targetOptions));
+            stats.trackPutObjectOperation();
```

Comment:
As mentioned below, let's give this a separate key to track under so we get metrics on what upload type we use how often.